test/unit/org/openstreetmap/josm/data/protobuf/ProtobufPackedTest.java

```diff
@@ -13,15 +11,15 @@
 class ProtobufPackedTest {
     @Test
     void testSingleByteNumbers() {
-        long[] numbers = new ProtobufPacked(new ByteArrayOutputStream(), ProtobufTest.toByteArray(new int[]{0, 0, 1, 1, 2, 2, 3, 3, 4, 4}))
+        long[] numbers = new ProtobufPacked(ProtobufTest.toByteArray(new int[]{0, 0, 1, 1, 2, 2, 3, 3, 4, 4}))
```

Comment:
This is fair. I haven't removed the deprecated code yet, but I think I was the only person to ever use it outside of JOSM core.